python-threatexchange/threatexchange/cli/hash_cmd.py

```diff
@@ -79,6 +80,26 @@ def init_argparse(cls, settings: CLISettings, ap: argparse.ArgumentParser) -> No
             help="only generate these signal types",
         )
 
+        ap.add_argument(
+            "--preprocess",
```

Comment:
For the photo being a subcommand, how would I make it so that I can accept the arguments without having them processed as multiple files for the photo hash?

I will add the help as well and focus on making it more safe with the targeted subcommands.

Reply:
For this PR, let's go with what you've written